assets/js/cash-calculator.js

```diff
@@ -0,0 +1,71 @@
+const firstValueInput=document.getElementById('first-value');
+const secondValueInput=document.getElementById('second-value');
+const operationSymbol=document.getElementById('operation-symbol');
+const calcButton=document.getElementById('calc-button');
+const resultValue=document.getElementById('result');
+
+console.log('addawdw');
+
+const object={
+    firstValue:firstValueInput.value,
+    secondValue:secondValueInput.value,
+    operationSymbolValue:operationSymbol.value,
+    result: '',
+    calculationHistory:[],
+    switchResult: function (){
+        switch (this.operationSymbolValue){
+            case'+':return Number(this.firstValue)+Number(this.secondValue);
+            case'-':return Number(this.firstValue)-Number(this.secondValue);
+            case'*':return Number(this.firstValue)*Number(this.secondValue);
+            case'/':return Number(this.firstValue)/Number(this.secondValue);
+        }
+    },
+    calculation: function (){
+        let cashCheck=this.calculationHistory.find(exc=>{
```

Comment:
_Code performance_: consider defining the `calculationHistory` as an object with calc expressions as keys, rather than an array.

Reply:
fixed